OCPBUGS-32475: Remove direct AI OCI steps from RH docs (#74877)
openshift/openshift-docs: modules/using-assisted-installer-oci-agent-iso.adoc

:_mod-docs-content-type: PROCEDURE
[id="using-assisted-installer-oci-agent-iso_{context}"]
= Using the {ai-full} to generate an {oci}-compatible discovery ISO image

Generate a discovery ISO image and upload the image to {oci-first}, so that the agent can perform hardware and network validation checks before you install an {product-title} cluster on {oci}.

From the {oci} web console, you must create the following resources:

* A compartment for better organizing, restricting access, and setting usage limits to {oci} resources.

> * An object storage bucket for safely and securely storing the discovery ISO image. You can access the image at a later stage for the purposes of booting the instances, so that you can then create your cluster.

Reviewer: > a later stage

Later stage of what? Or, perhaps this is understood as part in the OCI context??

Author: Yes, this becomes clearer further on in the integrated workflow. It's now in the Oracle docs so general references are safer for better doc maintainability on our side.

![Screenshot from 2024-05-01 15-33-35](https://github.com/openshift/openshift-docs/assets/57954076/6b39833d-9784-4119-af23-19f3df976945)

.Prerequisites

> * You created a child compartment and an object storage bucket on {oci}. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-assisted.htm#provisioning-cloud-infrastructure[Provisioning Cloud Infrastructure ({oci} Console)] in the _Oracle_ documentation.

Reviewer: ```suggestion
* You created a child compartment and an object storage bucket on {oci}. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-assisted.htm#provisioning-cloud-infrastructure[Provisioning Cloud Infrastructure ({oci} Console)] in the Oracle documentation.
```

* You reviewed details about the {product-title} installation and update processes.
* If you use a firewall and you plan to use a Telemetry service, you configured your firewall to allow {product-title} to access the sites required.

> * Before you create a virtual machines (VM), reference {op-system-base-full} certified shapes and instance types to identify the supported {oci} VM shapes. For more information about supported Oracle virtual machines, see link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types (Red Hat Ecosystem Catalog portal)].

Reviewer: I know zero about Oracle shapes and images, but this sentence still sound awkward to me. Is this way off? 
```suggestion
* Before you create a virtual machines (VM), see link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types (Red Hat Ecosystem Catalog portal)] to identify the supported {oci} VM shapes.
```

.. Download the file to your local system.

. Upload the discovery ISO image to the {oci} bucket. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-assisted.htm#install-cluster-upload-iso[Uploading The Red Hat ISO Image to Object Storage (Oracle documentation)].

> .. You must create a pre-authenticated request for your uploaded discovery ISO image. Ensure that you copy the URL from the pre-authenticated request, because you must specify the URL at a later stage when you create an {oci} stack.

Reviewer: ```suggestion
.. You must create a pre-authenticated request for your uploaded discovery ISO image. Ensure that you make note of the URL from the pre-authenticated request, because you must specify the URL at a later stage when you create an {oci} stack.
```
